### PR TITLE
Docs updates tf

### PR DESCRIPTION
**Pull request by ometelytsia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,16 +5,16 @@ This repository contains Taurus scripts for performance testing of Atlassian Dat
...
-    * Jira [Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html): `8.20.13`, `9.1.0`
+    * Jira [Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html): `8.20.13` and `9.1.0` platform release
 
 * Supported Jira Service Management versions: 
-    * Jira Service Management [Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html): `4.20.13`, `5.1.0`
+    * Jira Service Management [Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html): `4.20.13` and `5.1.0` platform release
     
 * Supported Confluence versions:
-    * Confluence [Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html): `7.13.7`, `7.19.0`
+    * Confluence [Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html): `7.19.2`, `7.13.7`
```
Comment (dmika1) [inline README.md:14]: I think versions should be in ascending order

File: README.md [hunk 2]
```diff
@@ -5,16 +5,16 @@ This repository contains Taurus scripts for performance testing of Atlassian Dat
...
-    * Jira Service Management [Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html): `4.20.13`, `5.1.0`
+    * Jira Service Management [Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html): `4.20.13` and `5.1.0` platform release
     
 * Supported Confluence versions:
-    * Confluence [Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html): `7.13.7`, `7.19.0`
+    * Confluence [Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html): `7.19.2`, `7.13.7` 
 
 * Supported Bitbucket Server versions:
-    * Bitbucket Server [Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html): `7.17.11`, `7.21.5` and `8.0.4` platform release. 
+    * Bitbucket Server [Long Term Support release](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html): `7.21.5`, `7.17.11`, and `8.0.4` platform release.
```
Comment (dmika1) [inline README.md:17]: I think versions should be in ascending order

**Review by opopovss — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ometelytsia) [reply README.md:14]: Moved into "recommended" order.

File: README.md (hunk 2 above)
Comment (ometelytsia) [reply README.md:17]: The same here, organized into recommended order: latest LTS, previous LTS, platform release.

**Pull request merged**